media-gfx/hydrus/hydrus-511.ebuild

```diff
@@ -0,0 +1,149 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Please try to push `EAPI=8` ebuilds whenever possible. I see hydrus-505 is already on EAPI-8, most likely updated by me. But don't downgrade on future version bumps without a reason.